_posts/2017/2017-04-06-javaland-2017-in-phantasialand.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "JavaLand 2017 - Welcome to Phantasialand"
+date: "2017-04-06 15:16:17"
+image: blog-header/microxchg-2017.jpg
+categories: events
+authors: ["Benjamin, Bastian"]
```

Comment:
is Bastian co-author or is it just you? Because you use first person perspective throughout the post